src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString.split(';');
+  const obj = {};
+
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i].trim()) {
+      const key = arr[i].split(':')[0].trim();
+      const value = arr[i].split(':')[1].trim();
```

Comment:
Alternative variant writing ( example for view )

```suggestion
      const [ key, value ] = arr[i].split(':').trim();
```